realsense2_camera/src/base_realsense_node.cpp

```diff
@@ -520,6 +522,12 @@ void BaseRealSenseNode::frame_callback(rs2::frame frame)
             if (f.is<rs2::video_frame>())
                 ROS_DEBUG_STREAM("frame: " << f.as<rs2::video_frame>().get_width() << " x " << f.as<rs2::video_frame>().get_height());
 
+            if (f.is<rs2::labeled_points>())
```

Comment:
better change this if statement to else if - do you agree?

Reply:
done